roboteam_ai/src/skills/Skill.cpp

```diff
@@ -15,11 +15,20 @@ void Skill::publishRobotCommand() {
     std::string ourSideParam;
     nh.getParam("our_side", ourSideParam);
 
-//    if(ourSideParam=="right"){
-//        cmd=rotateRobotCommand(cmd);
-//    }
-    ioManager.publishRobotCommand(command); // We default to our robots being on the left if parameter is not set
+    if(Constants::GRSIM() && ourSideParam=="right"){
```

Comment:
This should be done in a seperate pr right? Why is this in here?

Reply:
this is now in development. needed it for 2 teams